Neos.Fusion.Afx/composer.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "neos/fusion-afx",
+  "description": "JSX inspired compact syntax for Neos.Fusion",
+  "type": "neos-package",
+  "license": "GPL-3.0-or-later",
+  "replace": {
+    "packagefactory/atomicfusion-afx": "*"
+  },
+  "require": {
+    "neos/fusion": "^4.2 || ^5.0 || dev-master"
```

Comment:
Needs to be `|| ^6.0` :slot_machine:😉 too then. Or we go with `"*"` or `"self.version"` as in some of the other packages.

Reply:
Good point!

Reply:
I agree that this should be '*' or 'self.version'

Reply:
Yep, right